### ISSUE TITLE
RuntimeError: No driver of supported ones ('sqlite2', 'sqlite3') is available

### ISSUE THREAD

**Issue by mahbubcubd** [web2py/py4web#362]
Traceback (most recent call last):
  File "/home/user1/misc/env/lib/python3.8/site-packages/pydal/base.py", line 507, in __init__
    self._adapter = adapter(**kwargs)
  File "/home/use1/misc/env/lib/python3.8/site-packages/pydal/adapters/__init__.py", line 41, in __call__
    obj = super(AdapterMeta, cls).__call__(*args, **kwargs)
  File "/home/use1/misc/env/lib/python3.8/site-packages/pydal/adapters/base.py", line 424, in __init__
    super(SQLAdapter, self).__init__(*args, **kwargs)
  File "/home/use1/misc/env/lib/python3.8/site-packages/pydal/adapters/base.py", line 85, in __init__
    self.find_driver()
  File "/home/use1/misc/env/lib/python3.8/site-packages/pydal/adapters/base.py", line 133, in find_driver
    raise RuntimeError(
RuntimeError: No driver of supported ones ('sqlite2', 'sqlite3') is available

I am running py4web in python3.8 virtual environment with uwsgi. It was working fine with the system python3.6 before. 
I am using CentOS 7

**Comment by mdipierro** [comment]
Somehow in your python3.8 sqlite3 is not installed. You may need to install it.
Maybe

python3.8 -m pip install pysqlit3 -U

**Comment by mahbubcubd** [comment]
Thanks for the help.
Firstly I was not able to install pysqlit3 and I found that the problem was with the python3.8 dev package ( **sqlite-devel** ). I had installed python3.8 from source since RedHat has no python3.8 support yet.

I have resolved dev packages and recompiled python3.8
here is the process

`sudo yum groupinstall 'development tools'`
```
sudo yum install bzip2-devel expat-devel gdbm-devel \
    ncurses-devel openssl-devel readline-devel wget \
    sqlite-devel tk-devel xz-devel zlib-devel libffi-devel
```
```
mkdir temp
cd temp
wget https://www.python.org/ftp/python/3.8.6/Python-3.8.6.tgz
tar xzf Python-3.8.6.tgz
cd Python-3.8.6
sudo ./configure --enable-optimizations
sudo make altinstall
```
and finally,

python3.8 -m pip install pysqlit3 -U

Well, I have created virtualenv again and haven't face any issue to install all necessary modules

But I found another issue. It is not about importing module. I will open another issue for it.
